serverless/cmd/client/client.go

```diff
@@ -64,11 +85,13 @@ func main() {
 		glog.Exitf("Invalid log URL: %q", err)
 	}
 
-	// TODO(al) derive this from log pub key
-	const logID = "test"
+	// Derive logID from log public key
+	v, _ := note.NewVerifier(pubKey)
+	key_hash := v.KeyHash()
```

Comment:
golang uses `camelCase` rather than `under_score` naming, but I think you can inline this away into the following line anyway?

Reply:
Done